ConfigData/phase GEMS Comparison.py

```diff
@@ -239,19 +249,68 @@ def GEMSChiPlot(AtPct, Protosolar):
     # print('xmean: ', xmean)
     # print('Chi Squared result from scipy:', ScipyChiSquare)
 
-
-    plt.figure(3)
-    plt.clf()
+    # FIXME
+    # TODO Approach that using mpld3, failed.
+    # fig = Figure(figsize=(5,4))
+    # ax = fig.subplots()
+    # IncludedZ = where(~isnan(Chondricity))[0]+1
+    # TickLabels = [El for Z, El in enumerate(pb.ElementalSymbols) if Z in IncludedZ]
+    # TickInds = list(range(len(TickLabels)))
+    # ax.scatter(TickInds, Chondricity[IncludedZ-1], marker='o', color='red', s=150, alpha=0.5, label='Chondricity')
+    # ax.scatter(TickInds, ElementSigma[IncludedZ-1], marker='o', color='blue', s=150, alpha=0.5, label='Elemental sigmas from mean')
+    # ax.set_xticks(TickInds, TickLabels, rotation='vertical')
+    # ax.axhline(0, 0, 92, color='green', linewidth=3, label='Chondritic/GEMSitic')
+    # ax.legend()
+    # ax.set_ylabel('Chondricity:\nlog$_{10}$(At%/Chondritic)\nVariation from GEMS ($\sigma$)', fontsize=FontSizeBasis)
+    # fig.tight_layout()
+    # mpld3.save_html(fig, "templates/b.html")
+
+
+    fig = Figure(figsize=(6,4.5))
+    ax = fig.subplots()
     IncludedZ = where(~isnan(Chondricity))[0]+1
     TickLabels = [El for Z, El in enumerate(pb.ElementalSymbols) if Z in IncludedZ]
     TickInds = list(range(len(TickLabels)))
+<<<<<<< HEAD
+    ax.scatter(TickInds, Chondricity[IncludedZ-1], marker='o', color='red', s=150, alpha=0.5, label='Chondricity')
+    ax.scatter(TickInds, ElementSigma[IncludedZ-1], marker='o', color='blue', s=150, alpha=0.5, label='Elemental sigmas from mean')
+    ax.set_xticks(TickInds, TickLabels, rotation='vertical')
+    ax.axhline(0, 0, 92, color='green', linewidth=3, label='Chondritic/GEMSitic')
+    ax.legend()
+    ax.set_ylabel('Chondricity:\nlog$_{10}$(At%/Chondritic)\nVariation from GEMS ($\sigma$)', fontsize=FontSizeBasis)
+    fig.tight_layout()
+
+    # Save it to a temporary buffer.
+    buf = BytesIO()
+    fig.savefig(buf, format="png")
+    # Embed the result in the html output.
+    data = base64.b64encode(buf.getbuffer()).decode("ascii")
+    plot2 = f"<img src='data:image/png;base64,{data}'/>"
+
+    return Chondricity, ElementSigma, GEMSChiSqRed, DOF, plot2
+
+    # This is the original one
```

Comment:
This is a previous merge you were supposed to sort out.  You choose either the HEAD part (if what was on the server was correct) or you choose the >>>>> checksum part if your own code was correct.  Or if both are wrong, you edit so it is correct.